Examples/Scripts/KF_timing.sh

```diff
@@ -51,20 +71,21 @@ echo "Number of tracks per event: ${numTracksPerEvent}"
 echo "Number of events: ${numEvents}"
 
 time_stamp=`date +%F%T`
-exe_dir=$PWD
+exe_dir=${PWD}
 run_dir=KalmanFitter_timing_${time_stamp}
+
 mkdir ${run_dir}
 cd ${run_dir}
 
 output_file='output.log'
-echo "*****KalmanFitter timing test vs. pT*****" > ${output_file}
```

Comment:
Would you rather have a `pT` version, I think that would make it clearer, but then we need to ask @msmk0 to change the input, I suppose.

Reply:
Exactly. Let's handle this after we have the `pT` input.

Reply:
I am not quite sure which input you are referring to here?

Reply:
It's the `--gen-p-gev` which was before `--gen-pt-range`, though I agree with @XiaocongAi - we can address this in another PR.